release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3209,3 +3209,19 @@ $ oc adm release info 4.12.3 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+// TODO: Include an update like this in whichever 4.12 z-stream this gets backported to:
+[id="ocp-4-12-x-features"]
+==== Features
+
+[id="ocp-4-12-x-psa-tp-feature-set"]
+===== Pod security admission restricted enforcement (Technology Preview)
+
+With this update, pod security admission restricted enforcement is available as a Technology Preview feature by enabling the `TechPreviewNoUpgrade` feature set. Now, if you enable the `TechPreviewNoUpgrade` feature set, pods are rejected if they violate pod security standards, instead of only logging a warning.
```

Comment:
can we highlight the "enforcement" here?

Reply:
Done!